packages/chain-helpers/tsconfig.esm.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.esm.json",
```

Comment:
Not 100% happy with this since it duplicates - so options were -

- don't add the root `tsconfig.esm.json`
- extend from the local and add the esm/nodeResolutions outputs to all

Neither approach is DRY, but I guess the approach specified here (which is not committed), may be DRY-er.